test/utils/update_resources.go

```diff
@@ -48,9 +48,16 @@ func ScaleResourceWithRetries(scalesGetter scaleclient.ScalesGetter, namespace,
 		Size:            -1,
 		ResourceVersion: "",
 	}
+	// timeouts per iterations: 5s, 10s, 20s, 40s...
+	exponentialBackoff := wait.Backoff{
```

Comment:
```suggestion
	const exponentialBackoff := wait.Backoff{
```

Reply:
It looks like we can be how to test things in this directory?

Reply:
I am unsure what do you ask. `exponentialBackoff` object is no used anywhere except in the call to `wait.ExponentialBackoff()`. IMHO it is a good practice to use `const` to emphasis that this variable is not intended for a change.
Anyway, it was proposed as a suggestion.  AFAIK it should not block you.

Reply:
Sure will add that

Reply:
@minherz can you LGTM and approve this pr?

Reply:
Once I have added the const

Reply:
![image](https://github.com/kubernetes/kubernetes/assets/65275144/b4dec98b-b49d-4666-8f2f-3c95c54f8013)

Reply:
> can you LGTM and approve this pr?

I do not have permissions to LGTM it :slightly_frowning_face: 

Regarding my comment about `const`, I apologies for confusing you. It was my honest mistake. Go does not support constant structure types.

Reply:
No issues 👍